README.md

```diff
@@ -49,7 +49,7 @@ import { SpotifyAuth, Scopes } from 'react-spotify-auth'
 import 'react-spotify-auth/dist/index.css'
 
 const App = () => {
-  const token = Cookies.get('spotifyAuthToken')
+  let [token, setToken] = useState('');
```

Comment:
```suggestion
    const [token, setToken] = React.useState(Cookies.get("spotifyAuthToken"))
```

Just wanna keep it copy-pastable with vanilla React + ability to get token from storage